Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v4.5.0
+- Adds new API which provides a way to determine the sign-in methods associated with an
+  email address.
+- Adds new API which allows authentication using only an email link (Passwordless Authentication).
```

Comment:
(passwordless authentication with email link) since phone number sign in is considered passwordless and federated sign-in in some context is also considered so.